src/main/java/appeng/block/networking/BlockCableBus.java

```diff
@@ -82,13 +82,14 @@
 {
 
 	private static final ICableBusContainer NULL_CABLE_BUS = new NullCableBusContainer();
-	public static Class<? extends TileEntity> noTesrTile;
-	public static Class<? extends TileEntity> tesrTile;
+	private static Class<? extends TileEntity> noTesrTile;
+	private static Class<? extends TileEntity> tesrTile;
+
 	/**
 	 * Immibis MB Support.
 	 */
-	boolean ImmibisMicroblocks_TransformableBlockMarker = true;
-	int myColorMultiplier = 0xffffff;
+	final boolean ImmibisMicroblocks_TransformableBlockMarker = true;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1170)